src/openscm_units/data/metric_conversions.csv

```diff
@@ -1,92 +1,92 @@
-Source,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf
-OpenSCMMetricName,SARGWP100,AR4GWP100,AR5GWP100
-OpenSCM species,,,
-CH4,21,25,28
-N2O,310,298,265
-CFC11,3800,4750,4660
-CFC12,8100,10900,10200
-CFC13,,14400,13900
-CFC113,4800,6130,5820
-CFC114,,10000,8590
-CFC115,,7370,7670
-Halon1301,5400,7140,6290
-Halon1211,,1890,1750
-Halon2402,,1640,1470
-Halon1202,,,
-CCl4,1400,1400,1730
-CH3Br,,5,2
-CH3CCl3,100,146,160
-HCFC21,,,148
-HCFC22,1500,1810,1760
-HCFC123,90,77,79
-HCFC124,470,609,527
-HCFC141b,600,725,782
-HCFC142b,1800,2310,1980
-HCFC225ca,,122,127
-HCFC225cb,,595,525
-HFC23,11700,14800,12400
-HFC32,650,675,677
-HFC41,150,,116
-HFC125,2800,3500,3170
-HFC134,1000,,1120
-HFC134a,1300,1430,1300
-HFC143,300,,328
-HFC143a,3800,4470,4800
-HFC152,,,16
-HFC152a,140,124,138
-HFC161,,,4
-HFC227ea,2900,3220,3350
-HFC236cb,,,1210
-HFC236ea,,,1330
-HFC236fa,6300,9810,8060
-HFC245ca,560,,716
-HFC245fa,,1030,858
-HFC365mfc,,794,804
-HFC4310mee,1300,1640,1650
-SO2F2,,,4090
-SF6,23900,22800,23500
-NF3,,17200,16100
-CF4,6500,7390,6630
-C2F6,9200,12200,11100
-C3F8,7000,8830,8900
-cC4F8,8700,10300,9540
-C4F10,7000,8860,9200
-C5F12,7500,9160,8550
-C6F14,7400,9300,7910
-C7F16,,,7820
-C8F18,,,7620
-C10F18,,,7190
-SF5CF3,,17700,17400
-cC3F6,,,9200
-HFE125,14900,12400,
-HFE134,6320,5560,
-HFE143a,756,523,
-HCFE235da2,350,491,
-HFE245cb2,708,654,
-HFE245fa2,659,812,
-HFE347mcc3,575,530,
-HFE347pcf2,580,889,
-HFE356pcc3,110,413,
-HFE449sl,297,421,
-HFE569sf2,59,57,
-HFE4310pccc124,1870,2820,
-HFE236ca12,2800,5350,
-HFE338pcc13,,1500,2910
-HFE227ea,,,6450
-HFE236ea2,,,1790
-HFE236fa,,,979
-HFE245fa1,,,828
-HFE263fb2,,,1
-HFE329mcc2,,,3070
-HFE338mcf2,,,929
-HFE347mcf2,,,854
-HFE356mec3,,,387
-HFE356pcf2,,,719
-HFE356pcf3,,,446
-HFE365mcf3,,,
-HFE374pc2,,,627
-PFPMIE,,10300,9710
-CHCl3,4,,16
-CH2Cl2,9,8.7,9
-CH3Cl,,13,12
-Halon1201,,,376
\ No newline at end of file
+Source,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,"https://www.ipcc.ch/site/assets/uploads/2018/07/WGI_AR5.Chap_.8_SM.pdf, Table 8.SM.16 and https://www.ipcc.ch/site/assets/uploads/2018/02/WG1AR5_Chapter08_FINAL.pdf , Table 8.7 (page 714)."
+OpenSCMMetricName,SARGWP100,AR4GWP100,AR5GWP100,AR5CCFGWP100
```

Comment:
Is `AR5CCFGWP100` a standard name? If no, can we add some more detail in the source e.g. a comment like "AR5CCFGWP100 is AR5GWP100 including climate-carbon cycle feedbacks"?

Reply:
I don't think it's a standard name. The acronym for climate-carbon cycle feedbacks used in AR5 for GWPs is "cc fb" (AR5 WG1, Table 8.7). AR5CCF is used in the PRIMAP emissions module, so I kept that.

Reply:
I've added a comment